polaris-react/src/components/Banner/Banner.scss

```diff
@@ -5,17 +5,13 @@
   background-color: var(--p-background);
 
   @if $in-page {
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    @include focus-ring('wide');
     border-radius: var(--p-border-radius-2);
 
     @media #{$p-breakpoints-sm-down} {
       border-radius: 0;
       border-width: var(--p-border-width-1) 0;
     }
   } @else {
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    @include focus-ring('base');
```

Comment:
the "wide" and "base" props change the border radius of the focus ring, but since I'm using outline now, it will take the existing border radius on the banner automatically